vertx-grpc-server/src/main/java/io/vertx/grpc/server/GrpcServerRequest.java

```diff
@@ -78,4 +79,9 @@ public interface GrpcServerRequest<Req, Resp> extends GrpcReadStream<Req> {
    * @return the underlying HTTP connection
    */
   HttpConnection connection();
+
+  User user();
+
+  GrpcServerRequest<Req, Resp> setUser(User user);
```

Comment:
we don't need this I think

Reply:
I removed the setter from the interface but the getter is required. Otherwise a user of the API is unable to access the authenticated user. At this point there is no "Context" object for gRPC so I put the user in the request.